### PR TITLE
Updated russian translation

### PR DESCRIPTION
**Pull request by e5faf2**
New translations for "unfinished". Also reviewed old translations.

### CODE REVIEW

**Comment by PeterSurda** [comment]
Great. I will review it. Would you be willing to work on the translations for the new UI as well? See https://github.com/mailchuck/PyBitmessage

**Comment by e5faf2** [comment]
Yep, I will send pull request for translation of new UI some later.

**Comment by PeterSurda** [comment]
I reviewed it and I think it's ok, but I have some comments.

Can you PGP sign your commit?

While my Russian is not very advanced, are you sure that words like "Нотификации" and "трей" are appropriate translations? Is that consistent with the OS localisation, don't these use more native alternatives? I see that you already made some changes to make the translations more appropriate, for example you changed "Форсировать" into "принудительно", so maybe you can briefly compare what terms are used in a Russian OS localisation?

I tried to apply it onto my fork, but it failed because the line numbers changed. Can you perhaps do a pull request against https://github.com/mailchuck/PyBitmessage so that I don't have to edit it myself?

Thanks.

File: src/translations/bitmessage_ru.ts [hunk 1]
```diff
@@ -937,92 +937,92 @@ p, li { white-space: pre-wrap; }
...
         <location filename="../bitmessageqt/__init__.py" line="3567"/>
         <source>There are no recent broadcasts from this address to display.</source>
-        <translation type="unfinished"></translation>
+        <translation>Нет последних сообщений для этой рассылки.</translation>
     </message>
     <message>
         <location filename="../bitmessageqt/__init__.py" line="3571"/>
         <source>Display the %1 recent broadcast from this address.</source>
-        <translation type="unfinished"></translation>
+        <translation>Отобразить %d последнее сообщение рассылки для этого адреса.</translation>
```
Comment (fanatid) [inline src/translations/bitmessage_ru.ts]: %1 ?

File: src/translations/bitmessage_ru.ts [hunk 2]
```diff
@@ -937,92 +937,92 @@ p, li { white-space: pre-wrap; }
...
     </message>
     <message>
         <location filename="../bitmessageqt/__init__.py" line="3575"/>
         <source>Display the %1 recent broadcasts from this address.</source>
-        <translation type="unfinished"></translation>
+        <translation>Отобразить %d последних сообщений рассылки для этого адреса.</translation>
```
Comment (fanatid) [inline src/translations/bitmessage_ru.ts]: %1 ?

File: src/translations/bitmessage_ru.ts [hunk 3]
```diff
@@ -937,92 +937,92 @@ p, li { white-space: pre-wrap; }
...
         <location filename="../bitmessageqt/__init__.py" line="3528"/>
         <source>Enter an address above.</source>
-        <translation type="unfinished"></translation>
+        <translation>Введите адрес в поле выше.</translation>
     </message>
     <message>
         <location filename="../bitmessageqt/__init__.py" line="3557"/>
         <source>Address is an old type. We cannot display its past broadcasts.</source>
-        <translation type="unfinished"></translation>
+        <translation>Адрес имеет устаревший тип. Невозможно отобразить сообщения, отправленные ранее.</translation>
```
Comment (fanatid) [inline src/translations/bitmessage_ru.ts]: excess comma

**Comment by fanatid** [comment]
@PeterSurda it's a good translate
@e5faf2 :+1:

**Comment by PeterSurda** [comment]
@fanatid Thanks for the info, I missed the %1/%d, D'oh.

**Comment by e5faf2** [comment]
Thanks for review, Kirill.

Отправлено с iPhone

**Pull request closed without merging**